groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -33,15 +33,10 @@ groups:
       - kmuldoon@vmware.com
       - kakaraparthy@vmware.com
       - christi.schlotter@gmail.com
-      # CAPI release-1.4 team members
-      - alexandr.demicev@suse.com
-      - amit9116260192@gmail.com
+      # CAPI release-1.5 team members
+      - muhammad.adil.ghaffar@est.tech
       - codewithaniruddha@gmail.com
-      - furkat.gofurov@est.tech
```

Comment:
I would be happy to be moved to the upper list (outside of RT members) to get the alerts constantly but with the new email (furkat.gofurov@suse.com)